LDMP/settings.py

```diff
@@ -301,6 +313,9 @@ def load_settings(self):
         if buffer_size:
             self.buffer_size_km.setValue(float(buffer_size))
         self.checkbox_buffer.setChecked(buffer_checked)
+        self.area_settings_name.setText(
+            self.settings.value("trends_earth/region_of_interest/area_settings_name", '')
```

Comment:
as far as I can see  there is a principal problem in this code is that if a use set a custom name, any area parameter modification will override custom string.
Would it make sense to add a checkbox locking the generated or custom string?

Reply:
@luipir I think that is the expected behaviour, also this is only executed once when opening up the settings dialog.

Reply:
ok